isb_web/main.py

```diff
@@ -252,6 +254,24 @@ async def login():
 """
 
 
```

Comment:
This is the new method -- @datadavev do you have a better suggestion for what the URL should be?